serverless.yaml

```diff
@@ -100,8 +100,7 @@ provider:
     restApi:
       format: '{"authorizer.claims.sub":"$context.authorizer.claims.sub","error.message":"$context.error.message","extendedRequestId":"$context.extendedRequestId","httpMethod":"$context.httpMethod","identity.sourceIp":"$context.identity.sourceIp","integration.error":"$context.integration.error","integration.integrationStatus":"$context.integration.integrationStatus","integration.latency":"$context.integration.latency","integration.requestId":"$context.integration.requestId","integration.status":"$context.integration.status","path":"$context.path","requestId":"$context.requestId","responseLatency":"$context.responseLatency","responseLength":"$context.responseLength","stage":"$context.stage","status":"$context.status"}'
       executionLogging: false
-      role: !GetAtt ApiGatewayRestApiCloudwatchLogsCustomRole.Arn
```

Comment:
I don't know why serverless has this field if it doesn't actually set the value. I guess it's a problem for never.

Reply:
I tried this `role` property. but I got a permission error. This property was expecting an existing role, and we cannot create a role and set it.
![Capture](https://user-images.githubusercontent.com/88723283/165856083-91649bd3-bc3a-41d2-a1fd-9e2628e4fefd.JPG)